frontend/src/App.js

```diff
@@ -100,81 +40,34 @@ class App extends Component {
-                    >
-                      <li className="bg-white" role="menuitem">
-                        <NavLink
-                          aria-current="page"
-                          isActive={isHomeActive}
-                          className="nav-link"
-                          to={PATH.home}
-                        >
-                          {LOCALIZE.mainTabs.homeTabTitle}
-                        </NavLink>
-                      </li>
-                      <li className="bg-white" role="menuitem">
-                        <NavLink
-                          aria-current="page"
-                          isActive={isDashboardActive}
-                          className="nav-link"
-                          to={PATH.dashboard}
-                        >
-                          {LOCALIZE.mainTabs.dashboardTabTitle}
-                        </NavLink>
-                      </li>
-                      <li className="bg-white" role="menuitem">
-                        <NavLink
-                          aria-current="page"
-                          isActive={isPrototypeActive}
-                          className="nav-link"
-                          to={PATH.prototype}
-                        >
-                          {LOCALIZE.mainTabs.prototypeTabTitle}
-                        </NavLink>
-                      </li>
-                      <li className="bg-white" role="menuitem">
-                        <NavLink
-                          aria-current="page"
-                          isActive={isStatusActive}
-                          className="nav-link"
-                          to={PATH.status}
-                        >
-                          {LOCALIZE.mainTabs.statusTabTitle}
-                        </NavLink>
-                      </li>
-                    </ul>
-                  </div>
-                )}
-                <div aria-label="login-button" className="fixed-top" style={styles.loginButton}>
-                  <LoginButton />
-                </div>
-                <div
-                  aria-label={LOCALIZE.ariaLabel.languageToggleBtn}
-                  className="fixed-top"
-                  style={styles.languageButton}
-                >
-                  <Translation />
-                </div>
-              </nav>
-            </header>
+            {!isTestActive && (
+              <Navbar bg="light" variant="light">
+                <Navbar.Brand href="/">
+                  <img alt="" src={canada_logo} width="30" className="d-inline-block align-top" />
+                  {"Public Service Commission"}
+                </Navbar.Brand>
+                <Nav className="mr-auto">
+                  <Nav.Link href="/">{LOCALIZE.mainTabs.homeTabTitle}</Nav.Link>
+                  <Nav.Link href="/emib-sample">{LOCALIZE.mainTabs.sampleTest}</Nav.Link>
+                  <Nav.Link href="/dashboard">{LOCALIZE.mainTabs.dashboardTabTitle}</Nav.Link>
```

Comment:
Should we put the **Dashboard** link before the **Sample test** link in order the following the design? Also, it seems like it is more logic to see the dashboard before doing the sample test, no?